src/patterns/dosdp-patterns/MarkersToCells.yaml

```diff
@@ -0,0 +1,44 @@
+pattern_name: NSForestMarkers
```

Comment:
I don't think this pattern is running at all. I couldn't see any terms generated by this template in the `definitions.owl`. I couldn't see any terms in my local build as well.
Errors are:
- missing column for pattern variable <Marker_set>
- missing column for pattern variable <Marker_set_xref>
- missing column for pattern variable <has_characterization_set>

Reply:
Quite likely,  I haven't looked closely yet.  Will try to find time tomorrow.